addons/controls/index.js

```diff
@@ -4,10 +4,10 @@
  * There is no `default` for this file.
  */
 
-export * from './configure';
+export * from './controls';
 export * from './boolean';
-export * from './button';
```

Comment:
sandstone/VideoPlayer of qa sampler used this button knob.
But I'm not seeing controls/button. What would be the replacement of this knob?

Reply:
Since there's no controls/button that acts as a knobs/button, I guess we could use controls/select for the replacement.
(supported controls list: https://storybook.js.org/docs/react/essentials/controls#annotation)